wallet/udb/upgrades.go

```diff
@@ -80,24 +80,30 @@ const (
 	// transactions.
 	hasExpiryFixedVersion = 9
 
+	// removeMultisigOutsWithoutTransactions is the tenth version of the database.
+	// It fixes issue with MultisigOuts without related transactions. Simply find all
+	// detached MultisigOutputs and remove
+	removeMultisigOutsWithoutTransactionsVersion = 10
```

Comment:
This name is a bit long.  `removeExtraMultisigOutsVersion` reads better to me.  The comment should also describe *why* they were an issue (failure to insert transactions due to multisig output double spend checks, because they were not always correctly removed in previous database code).  And please don't use "simply" when describing the fix, just describe how the fix is performed.